Cleanup of load tests

mangalpalli opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines: https://github.com/googleforgames/agones/blob/main/CONTRIBUTING.md and developer guide https://github.com/googleforgames/agones/blob/main/build/README.md
2. Please label this pull request according to what type of issue you are addressing.
3. Ensure you have added or ran the appropriate tests for your PR: https://github.com/googleforgames/agones/blob/main/build/README.md#testing-and-building
-->

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
>/kind bug

/kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

**What this PR does / Why we need it**:

After building container and running this command "docker run --rm --network="host" -e "LOCUST_FILE=gameserver_allocation.py" -e "TARGET_HOST=http://127.0.0.1:8001" -p 8089:8089 locust-files:latest", it is showing "missing /usr/local/bin/locust".

LOCUST files are no longer updated, and had to be removed.

Removed all the content related to LOCUST files.

**Which issue(s) this PR fixes**:
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Closes #<issue number>`, or `Closes (paste link of issue)`.
-->
Closes #2744

**Special notes for your reviewer**:

markmandel commented:
Is it possible to update #2779 instead of a new PR? Then we can keep the review history.

markmandel commented:
Never mind, I see you closed the other one.

gongmax commented on test/load/allocation/grpc/fixed.txt:
```diff
@@ -1,23 +0,0 @@
-# Copyright 2022 Google LLC All Rights Reserved.
```
We should also move this file under `test/load/allocation`

mangalpalli replied:
The file is ignored in the commit due to gitignore. Do I update the gitignore and push these files ?

gongmax replied:
The gitignore actually has statement to include those two files i.e.`!fixed.txt` and `!variable.txt`, but the order seems incorrect. I think to fix it we should put those two statements after `*.txt`, since what `!` does is "_any matching file excluded by a **previous** pattern will become included again_." [doc](https://git-scm.com/docs/gitignore).

mangalpalli replied:
Thanks, I've updated the PR.

markmandel commented:
Looks like you have some changes to install.yaml for some reason, which I don't think you need for this PR.

You may want to also rebase your PR against upstream/main on your PR branch, which will also cleanup your commit history as well.

mangalpalli commented:
> Looks like you have some changes to install.yaml for some reason, which I don't think you need for this PR.
> 
> You may want to also rebase your PR against upstream/main on your PR branch, which will also cleanup your commit history as well.

Hi @markmandel, I tried to pull the latest changes from main, and the changes reflected in the install.yaml file. After that I rebased the branch from main. I believe I should be reverting the commit which pushed the install.yaml file and that should resolve the build issue as well ?

markmandel commented:
> Hi @markmandel, I tried to pull the latest changes from main, and the changes reflected in the install.yaml file. After that I rebased the branch from main. I believe I should be reverting the commit which pushed the install.yaml file and that should resolve the build issue as well ?

First, I'd like to see this PR rebased against `main` -- so make sure your local `main` branch is up to date with upstream, and then rebase it against your local branch for this PR and you should end up with only commits for this PR. 

Then we can just work things out from there. Sound good?

mangalpalli commented:
> > Hi @markmandel, I tried to pull the latest changes from main, and the changes reflected in the install.yaml file. After that I rebased the branch from main. I believe I should be reverting the commit which pushed the install.yaml file and that should resolve the build issue as well ?
> 
> First, I'd like to see this PR rebased against `main` -- so make sure your local `main` branch is up to date with upstream, and then rebase it against your local branch for this PR and you should end up with only commits for this PR.
> 
> Then we can just work things out from there. Sound good?

Thanks @markmandel, I reverted the commit and rebased. It helped. I will be careful with the rebase and merge going ahead. As of now the build is succeeded, so you can review the PR and merge it, if there are no more changes.
